.github/workflows/test.yml

```diff
@@ -1,32 +1,74 @@
 on: push
-name: Run tests
+name: proj ci
 jobs:
-  proj:
-    name: proj
-    runs-on: ubuntu-latest
+  proj-ubuntu:
+    name: proj ubuntu
     if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        features: ["", "--features network", "--features bundled_proj", "--features \"bundled_proj network\""]
     container:
-      image: georust/geo-ci:latest
+      image: georust/proj-ci
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
-      - run: cmake --version
-      - run: cargo install cargo-all-features
-      - run: cargo build-all-features
```

Comment:
We can't currently use cargo-all-features because proj has mutually exclusive features (bundled_proj vs. pkg_config)

It's a little bit more verbose to spell out the feature combinations, but on the upside we get better parallelism in our test runners.

And for things like the macos builds, they're much slower, so I've only cherry-picked a couple of probably build configurations.